src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -130,27 +131,31 @@ function MoneyRequestPreviewContent({
     };
 
     const getPreviewHeaderText = (): string => {
+        let message = translate('iou.cash');
+
         if (isDistanceRequest) {
-            return translate('common.distance');
+            message = translate('common.distance');
+        }
+
+        if (isScanRequest) {
+            message = translate('tabSelector.scan');
         }
 
         if (isScanning) {
-            return translate('common.receipt');
+            message = translate('common.receipt');
         }
 
         if (isBillSplit) {
-            return translate('iou.split');
+            message = translate('iou.split');
         }
```

Comment:
Let's keep original message to avoid any regressions.
```suggestion
        if (isDistanceRequest) {
            message = translate('common.distance');
        } else if (isScanning) {
            message = translate('common.receipt');
        } else if (isBillSplit) {
            message = translate('iou.split');
        }
```